src/runtime/app.config.ts

```diff
@@ -700,6 +700,7 @@ const pagination = {
   wrapper: 'flex items-center -space-x-px',
   base: '',
   rounded: 'first:rounded-s-md last:rounded-e-md',
+  iconClass:'rtl:rotate-180',
```

Comment:
Not a huge fan of adding this here, could we find another way to achieve this?

Reply:
We can add on config for nexButton and prevButton to the icon by adding class, but this needs to change the v-bind for next and prev.

But If the application supports multilanguage, changing the icon in the config file will not work on other languages.

Or make a wrapper component for everyone who needs to use pagination and check if RTL adds a custom icon, which I don't love this way.

What do you think?

Reply:
@Haythamasalama I think it can send an icon in the right direction without reversing it

Reply:
@HassanKadhim 

I'm not really sure if I understand you.
Can you please provide an example?

Reply:
Maybe people who use RTL can just update the icon manually from their `app.config.ts`? Thing they would do anyway if they don't want to use Heroicons.

Reply:
> Maybe people who use RTL can just update the icon manually from their `app.config.ts`? Thing they would do anyway if they don't want to use Heroicons.

@benjamincanac 
But by this way, if the app supports multilingual languages, RLT will work, and LRT will not.

Reply:
You're making a point. Then, maybe adding a class attribute inside `prevButton` and `nextButton` with `rtl:[&_span]:rotate-180`?

Reply:
@benjamincanac 
Done 

![image](https://github.com/nuxtlabs/ui/assets/37311945/be6df6fd-bb56-4c20-b526-b429c4fc2e94)
![image](https://github.com/nuxtlabs/ui/assets/37311945/b35d8d16-40e7-411c-8694-491fc873dded)